src/ui/page/channel/view.jsx

```diff
@@ -67,22 +71,47 @@ function ChannelPage(props: Props) {
   const [editing, setEditing] = useState(false);
   const [thumbPreview, setThumbPreview] = useState(thumbnail);
   const [coverPreview, setCoverPreview] = useState(cover);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [searchResults, setSearchResults] = useState(undefined);
 
   // If a user changes tabs, update the url so it stays on the same page if they refresh.
   // We don't want to use links here because we can't animate the tab change and using links
   // would alter the Tab label's role attribute, which should stay role="tab" to work with keyboards/screen readers.
   const tabIndex = currentView === ABOUT_PAGE || editing ? 1 : 0;
-  const onTabChange = newTabIndex => {
+  function onTabChange(newTabIndex) {
     let url = formatLbryUriForWeb(uri);
     let search = '?';
     if (newTabIndex !== 0) {
       search += `${PAGE_VIEW_QUERY}=${ABOUT_PAGE}`;
     } else {
+      setSearchResults(null);
       search += `page=${page}`;
     }
 
     history.push(`${url}${search}`);
-  };
+  }
+
+  function handleSearch() {
```

Comment:
This should probably be in https://github.com/lbryio/lbry-redux/blob/master/src/redux/actions/search.js

Reply:
I initially thought that and I think it makes sense to put it there eventually, but I'm starting to think small stuff like this is fine if live in the components. 

All it is is one lighthouse call